lib/src/components/NxProgressBar/NxProgressBar.scss

```diff
@@ -40,25 +39,36 @@ $-progress-bar-color: var(--nx-swatch-teal-60);
   }
 
   &--success {
-    color: var(--nx-color-alert-success);
+    color: var(--nx-color-progress-bar-text-success);
+
+    .nx-icon {
+      color: var(--nx-color-progress-bar-icon-success);
+    }
 
     .nx-progress-bar__counter {
-      border-color: var(--nx-color-alert-success);
-      background-color: var(--nx-swatch-green-95);
+      border-color: var(--nx-color-progress-bar-counter-border-success);
+      background-color: var(--nx-color-progress-bar-counter-background-success);
     }
   }
 
   &--error {
-    color: var(--nx-color-alert-error);
+    color: var(--nx-color-progress-bar-text-error);
+
+    .nx-icon {
+      color: var(--nx-color-progress-bar-icon-error);
+    }
 
     .nx-progress-bar__counter {
-      border-color: var(--nx-color-alert-error);
-      background-color: var(--nx-swatch-red-95);
+      border-color: var(--nx-color-progress-bar-counter-border-error);
+      background-color: var(--nx-color-progress-bar-counter-background-error);
     }
   }
 }
 
 .nx-progress-bar__counter {
+  background-color: var(--nx-color-progress-bar-counter-background);
+  border-color: var(--nx-color-progress-bar-counter-border);
+  color: var(--nx-color-progress-bar-counter-percent);
```

Comment:
I suspect the intent is that the default counter here just follow the `nx-counter` styles; which it does without these properties.  The mockup does have the text color slightly different as @EmilyBermudez88 mentioned - black instead of grey-5 in light mode, but I'm willing to bet that's a mockup mistake.

Reply:
> I suspect the intent is that the default counter here just follow the `nx-counter` styles; which it does without these properties. The mockup does have the text color slightly different as @EmilyBermudez88 mentioned - black instead of grey-5 in light mode, but I'm willing to bet that's a mockup mistake.

I reached out to Jason and it's a mockup mistake

Reply:
> I reached out to Jason and it's a mockup mistake

Is it going to be fixed in the mockup? It's still black

Reply:
> > I reached out to Jason and it's a mockup mistake
> 
> Is it going to be fixed in the mockup? It's still black

I spoke to Jason about it. It's fixed now: https://www.sketch.com/s/2393264e-1233-4909-99c8-a7dda02588cd/a/m1vOaY7#Inspect